Makefile

```diff
@@ -60,7 +60,7 @@ release:
   	  -DBUILD_LAME="${BUILD_LAME}"             \
   	  -DCMAKE_SKIP_RPATH="${NO_RPATH}"     ..; \
       ${BUILD_SYSTEM} lrelease;                             \
-      ${BUILD_SYSTEM} -j ${CPUS};                           \
+      ${BUILD_SYSTEM} -l ${CPUS};                           \
```

Comment:
Downside: unfortunatly on Windows it by far doesn't use as much resources as it could (as much as -j does), so takes longer a fair amount